wremnants/HDF5Writer.py

```diff
@@ -190,12 +190,14 @@ def write(self,
             hist_nominal = dg.groups[procs_chan[0]].hists[chanInfo.nominalName] 
             hist_axes[chan] = [hist_nominal.axes[a] for a in axes]
 
+            hist_nominal_perProc = {}
             # nominal predictions
             for proc in procs_chan:
                 logger.debug(f"Now  in channel {chan} at process {proc}")
-                
+
                 # nominal histograms of prediction
                 norm_proc_hist = dg.groups[proc].hists[chanInfo.nominalName]
+                hist_nominal_perProc[proc] = norm_proc_hist.copy()
```

Comment:
This line is not needed if you remove the lines L290/291 below
```
            for proc in procs_chan:
                del dg.groups[proc].hists[chanInfo.nominalName]
```
and later the nominal histogram can just be accessed with ```dg.groups[proc].hists[chanInfo.nominalName]```